projects/auth0-angular/src/lib/auth.config.ts

```diff
@@ -171,6 +171,11 @@ export interface AuthConfig {
    */
   httpInterceptor?: HttpInterceptorConfig;
 
+  /**
+   * Path in your application to redirect to when the Authorization server
+   * returns an error. Defaults to `/`
+   */
+  errorPath?: string;
```

Comment:
Nitpick: can we add whitespace under here to separate from the comment block underneath?